### ISSUE TITLE
Uncaught TypeError: Cannot read property 'firstDayOfWeek' of null

### ISSUE THREAD

**Issue by pauleonardo** [jquense/react-widgets#667]
Hi, i can't resolve this error. These are the libraries versions that i used.

- Moment@2.19.1
- React-widgets@4.0.2
- React-widgets-moment@4.0.2

![image](https://user-images.githubusercontent.com/9327820/31746618-49edfcfc-b436-11e7-8115-dc88eb0aefcd.png)

**Comment by jquense** [comment]
seems like maybe you don't have a moment locale setup? or are trying to use a culture/locale that isn't loaded

**Comment by QuincyHe** [comment]
I got the same problem.
Using `moment 2.22.2` with ES6 (Webpack + ReactJS)
I'm pretty sure the relative require in `moment/lib/src/locale/locales.js`
```
var aliasedRequire = require;
aliasedRequire('./locale/' + name);
```
is fixed by using the ContextReplacementPlugin:

```
plugins: options.plugins.concat([
    ...
    new webpack.ContextReplacementPlugin(/\.[\\/]locale$/, '../../locale', /^\.\/(zh-cn|jp)$/),
    ...
]),
```
Strangely enough, I found that in `moment/lib/src/locale/locales.js`, line 51 and 52:
```
if (!locales[name] && (typeof module !== 'undefined') &&
            module && module.exports) {
```
`module.exports` is always `undefined`. If I remove `&& module.exports`. The block gets executed and loads 'zh-cn' locale file successfully.
![1528274521761](https://user-images.githubusercontent.com/17258180/41027376-b2bb5978-692b-11e8-98fe-b9af34c4d673.jpg)
I don't know why `moudle.exports` is undefined though.
